### PR TITLE
Changes and additions for method-level RPS

### PR DESCRIPTION
**Pull request by mmk-1**
This PR adds 4 Mojos which are used for method-level RPS. Each one has a counterpart  for the class-level RPS which is already available.
- Some changes in emop-maven-plugin/pom.xml like updating maven-plugin-plugin which is required for STARTS to run with the latest additions in [this](https://github.com/TestingResearchIllinois/starts/pull/106) PR for STARTS.
- Addition of required dependencies in pom.xml needed for finding method-to-spec mapping.
- MethodHelper.java in utils/ which is required for both method-level and hybrid (seperate PR for later) RPS.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -37,6 +37,17 @@
...
+      <groupId>org.jboss.forge.roaster</groupId>
+      <artifactId>roaster-api</artifactId>
+      <version>2.22.2.Final</version>
+  </dependency>
+  <dependency>
+      <groupId>org.jboss.forge.roaster</groupId>
+      <artifactId>roaster-jdt</artifactId>
+      <version>2.22.2.Final</version>
+      <scope>runtime</scope>
+  </dependency>
```
Comment (pengyuejiang) [inline pom.xml]: Please fix the indentations.

File: emop-maven-plugin/src/main/resources/META-INF/maven/lifecycle.xml [hunk 2]
```diff
@@ -111,4 +155,4 @@
       </phase>
     </phases>
   </lifecycle>
-</lifecycles>
+</lifecycles>
```
Comment (pengyuejiang) [inline emop-maven-plugin/src/main/resources/META-INF/maven/lifecycle.xml]: Please add a new line at the end of the file.

File: emop-core/src/main/java/edu/cornell/emop/util/MethodsHelper.java [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+    // test use
+    public static void putclassTomethod(String key, ArrayList<String> value) {
+        classToMethods.put(key, value);
+    }
+
+    // test use
+    public static void putmethodsToLineNumbers(String key, ArrayList<Integer> value) {
+        methodsToLineNumbers.put(key, value);
+    }
```
Comment (pengyuejiang) [inline emop-core/src/main/java/edu/cornell/emop/util/MethodsHelper.java]: Is there a better way to test these methods? One possible suggestion is using reflection.

**Review by pengyuejiang — CHANGES_REQUESTED** [review]
Just address these small comments. Thank you for the work!

**Comment by mmk-1** [comment]
Hello, I've addressed your comments, apologies for the delay!

File: scripts/setup.sh [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+# Globals:
+#   ENV_DIR
+#######################################
+function setup_emop {
+  if [ ! -d ${ENV_DIR}/emop ]; then
+    (
+      cd ${ENV_DIR}
+      git clone https://github.com/SoftEngResearch/emop
+      (
+	      cd emop
```
Comment (owolabileg) [inline scripts/setup.sh:129]: minor: fix spacing of that cd command

**Comment by owolabileg** [comment]
@pengyuejiang please comment quickly on this PR, just a quick "merge" or "no-merge" will do.

**Pull request merged**